alcs-frontend/src/app/features/admin/decision-condition-types/decision-condition-types-dialog/decision-condition-types-dialog.component.html

```diff
@@ -22,16 +22,104 @@ <h4>{{ isEdit ? 'Edit' : 'Create' }} Decision Condition Type</h4>
     <div class="full-width">
       <mat-form-field class="label" appearance="outline">
         <mat-label>Label</mat-label>
-        <input required matInput id="label" [(ngModel)]="label" name="label" />
+        <input required matInput id="label" formControlName="label" name="label" />
       </mat-form-field>
     </div>
 
     <div class="full-width">
       <mat-form-field class="description" appearance="outline">
         <mat-label>Description</mat-label>
-        <input required matInput id="description" [(ngModel)]="description" name="description" />
+        <input required matInput id="description" formControlName="description" name="description" />
       </mat-form-field>
     </div>
+
+    <div class="condition-fields-container full-width">
+      <span>Click to select fields</span>
+
+      <div class="condition-field">
+        <mat-checkbox formControlName="isComponentToConditionChecked">Component to Condition*</mat-checkbox>
+      </div>
+
+      <div class="condition-field">
+        <div class="condition-field-header">
+          <div>
+            <mat-checkbox formControlName="isAdministrativeFeeAmountChecked">Administrative Fee Amount</mat-checkbox>
+          </div>
+
+          <div class="toggle" *ngIf="conditionTypeForm.get('isAdministrativeFeeAmountChecked')?.value">
```

Comment:
nitpick: `conditionTypeForm.controls.isAdministrativeFeeAmountChecked?.value` is a little easier to read (to me anyway).